### DISCUSSION TITLE
Cannot load information for github

### DISCUSSION THREAD

**Discussion by DevAnabKhan** [community/community#128205] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

When I try to share my code to GitHub from Android Studio, I receive the following error message:

"Cannot load information for github.com/devAnabKhan: Request response. Access to this site has been restricted. If you believe this is an error, please contact https://support.github.com/."

All the functionality was working fine previously, but for the past two days, this message has been appearing. How can I resolve this issue?

**Comment by BinaryBreaker** [comment]
facing the same issue

**Reply by YaroSK26** [reply]
❤️

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @DevAnabKhan ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by hombrevago** [reply]
HI.  I'm having the same problem as of today.  worked fine yesterday.

also @queenofcorgis the Accessibility category refers to access by people with disabilities

**Comment by hombrevago** [comment]
this update resolved the issue for me.

Android Studio Jellyfish | 2023.3.1 Patch 2
Build #AI-233.14808.21.2331.11926650, built on June 5, 2024

**Comment by adfleshner** [comment]
I am seeing the exact same issue for a new project that is built in.
Android Studio Iguana 2023.2.1 Patch 2
Build #AI-232.10300.40.2321.11668458, built on April 3, 2024

I authorized the AS with github and when it went back to the IDE I saw the message above for my account.

**Comment by opasvo** [comment]
Same issue, but for IntelliJ IDEA
IntelliJ IDEA 2023.3.4 (Community Edition)
Build #IC-233.14475.28, built on February 13, 2024

**Reply by PabitraM29** [reply]
Yes I am too facing this issue

**Comment by EasyCodeCoach** [comment]
Response #10098

**Comment by Dineshprabha** [comment]
Did anyone get the solution for this issue?

**Reply by Dineshprabha** [reply]
This update solved the issue

Android Studio Koala | 2024.1.1 june 13, 2024

**Reply by ahmedNaser7** [reply]
lol i face this issue also with koala version

**Comment by pcox3** [comment]
Facing same for Android Studio Jellyfish | 2023.3.1 Patch 1

**Comment by Igor755** [comment]
WTF, not work Android Studio Jellyfish | 2023.3.1 Patch 1

**Comment by ricky-kiva** [comment, marked as answer]
**My solution**:

I fixed this by updating Android Studio (to be specific, to Android Studio Koala, 2024.1.1). Some people in the forum also fixed this by simply reinstalling the Android Studio (if you still have the installer)

**Comment by Igor755** [comment]
It's just a great solution to reinstall studio :-)

**Comment by IzmaIs** [comment]
I don't think that reinstalling the studio should solve this problem 😉😉🤣🙄🙄 After all, it doesn’t appear because of an update (at least I didn’t update WebStorm 😐😐😐

**Comment by kristal374** [comment]
Likewise, PyCharm 2022.3 (Professional Edition) and PyCharm 2024.1 (Professional Edition). Updating to the latest version helped me.

**Comment by madhusankha90** [comment]
Update your Android Studio to latest version..

**Comment by vbresan** [comment]
The workaround would be first to set the project's version control system to Git and then commit the project locally. When you decide to push it, you will have an option to set a remote URL. Create the repository on GitHub and paste the URL in Android Studio. And you are done!

**Comment by raizakanwal** [comment]
![Untitled](https://github.com/user-attachments/assets/bc1094bb-254c-4b02-a69d-5890fa98d1d7)

**Reply by raizakanwal** [reply]
How can I resolve this issue.

**Reply by kebora** [reply]
Upgrade your software, by downloading the latest version.

**Comment by chiou711** [comment]
Maybe GitHub Desktop is a workaround if you just need to create a GitHub repository first

**Comment by NodexSystem** [comment]
facing the same issue
